### DISCUSSION TITLE
Traxxas instead of Redcat Racing

### DISCUSSION THREAD

**Discussion by FredericPrecioso** [prl-mushr/mushr#80] [Hardware]
Hello,

It is very difficult to find Redcat Racing in France and even to order it abroad.
Is it transparent to replace Redcat Racing by a Traxxas brushless car?

It would be very simpler for us.

Thank you very much for your help.

Best regards,
Frederic

**Comment by palanc** [comment]
Hi @FredericPrecioso - We only support the Redcat models listed in our bill of materials. You could try using a Traxxas car but you would likely need to modify some of the 3D printed parts. If you're interested in doing that, you can find the existing OpenScad files that you would want to edit here: [https://github.com/prl-mushr/mushr_cad/tree/master/v3/scad](https://github.com/prl-mushr/mushr_cad/tree/master/v3/scad)

**Reply by FredericPrecioso** [reply]
Thank you for your answer.
